services/proxy/pkg/staticroutes/staticroutes.go

```diff
@@ -0,0 +1,54 @@
+package staticroutes
+
+import (
+	"github.com/go-chi/chi/v5"
+	"github.com/owncloud/ocis/v2/ocis-pkg/log"
+	"github.com/owncloud/ocis/v2/ocis-pkg/oidc"
+	"github.com/owncloud/ocis/v2/services/proxy/pkg/config"
+	microstore "go-micro.dev/v4/store"
+	"net/http"
+	"net/url"
+	"path"
+)
+
+// StaticRouteHandler defines a Route Handler for static routes
+type StaticRouteHandler struct {
+	Prefix         string
+	Proxy          http.Handler
+	UserInfoCache  microstore.Store
+	Logger         log.Logger
+	Config         config.Config
+	OidcClient     oidc.OIDCClient
+	OidcHttpClient *http.Client
+
+	oidcURL *url.URL
+}
+
+type jse struct {
+	Error            string `json:"error"`
+	ErrorDescription string `json:"error_description"`
+}
+
+func (s *StaticRouteHandler) Handler() http.Handler {
+	s.oidcURL, _ = url.Parse(s.Config.OIDC.Issuer)
+	s.oidcURL.Path = path.Join(s.oidcURL.Path, wellKnownPath)
+	m := chi.NewMux()
+	m.Route(s.Prefix, func(r chi.Router) {
+
+		// Wrapper for backchannel logout
+		r.Post("/backchannel_logout", s.backchannelLogout)
+
+		// openid .well-known
+		if s.Config.OIDC.RewriteWellKnown {
+			r.Get("/.well-known/openid-configuration", s.OIDCWellKnownRewrite)
```

Comment:
AFAICT s.backchannelLogout and s.OIDCWellKnownRewrite are be dedicated handlers that can be imported without the StaticRouteHandler. Either Upcase both or lowercase both.